insights/parsers/aws_instance_type.py

```diff
@@ -39,16 +39,20 @@ class AWSInstanceType(CommandParser):
     """
```

Comment:
Also, I guess the example on the line 20 should include the `-s` option.

Reply:
Yeah, good catch, I'll fix that.